services/workflows-service/src/business-report/business-report.repository.ts

```diff
@@ -65,4 +65,13 @@ export class BusinessReportRepository {
       ),
     );
   }
+
+  async count<T extends Omit<Prisma.BusinessReportFindManyArgs, 'where'>>(
```

Comment:
Not sure if the omit/where thing needs changing.
```suggestion
  async count<T extends Omit<Prisma.BusinessReportCountArgs, 'where'>>(
```